appConfig/appConfig.snd4.yaml

```diff
@@ -8,4 +8,6 @@
 - key: FILE_RECEIVER_TOPIC_ADDRESS
   value: "fcp-sfd-file-metadata"
 - key: FILE_RECEIVER_SUBSCRIPTION_ADDRESS
-  value: "fcp-sfd-file-receiver"
\ No newline at end of file
+  value: "fcp-sfd-file-receiver"
+- key: COSMOS_ENDPOINT
+  value: "https://fcp-sfd-temp-cosmos.documents.azure.com:443/"
```

Comment:
You'll be glad to know the files database (`ffc-sfd-customer-receiver-files`) and container (`files-container`) are up and running on our temporary Cosmos instance on SND4! 🤩 You can see it via the [Cosmos DB Data Explorer on the Azure portal](https://portal.azure.com/#@Defra.onmicrosoft.com/resource/subscriptions/da83801b-f7e0-4510-9848-67d87b1009e2/resourceGroups/sndadpinfrg4401-fcp-sfd-temp/providers/Microsoft.DocumentDb/databaseAccounts/fcp-sfd-temp-cosmos/dataExplorer).